integreat_cms/cms/models/media/media_file.py

```diff
@@ -97,20 +97,35 @@ def upload_path_thumbnail(instance, filename):
     return path
 
 
+
+def file_size_limit(value): 
+
+        limit = 3 * 1024 * 1024
+        if value.size > limit:
+            raise ValidationError('File too large. Size should not exceed 3 MiB.')
```

Comment:
```suggestion
        raise ValidationError(
            _("File too large. Size should not exceed {}.").format(
                filesizeformat(settings.MEDIA_MAX_UPLOAD_SIZE)
            )
        )
```
Could you make the text translatable and set the German string in the [translation file](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/locale/de/LC_MESSAGES/django.po) to:

>Datei zu groß. Die Größe sollte 3 MiB nicht überschreiten.

Also, I think it makes sense to use `filesizeformat()` to dynamically show the file size limit in case it is changed in the future.